webapp/Connector/src/app/view/module/DataAvailabilityModule.js

```diff
@@ -186,6 +186,26 @@ Ext.define('Connector.view.module.DataAvailabilityModule', {
         this.callParent();
     },
 
+    getInstructions: function (instructions) {
+        var instrWithHyperlink = instructions;
+        if (instructions) {
+            if (instrWithHyperlink.indexOf("Monoclonal antibodies") >= 0) {
+                var splitInstr = instrWithHyperlink.split("Monoclonal antibodies");
+                instrWithHyperlink = splitInstr[0] + " <a href=\"#mabgrid\" target=\"_blank\">Monoclonal antibodies</a>" + splitInstr[1];
```

Comment:
I don't think opening in new tab is the common practice for DataSpace. Usually, only external links are opened in new tabs.

Reply:
I don't think the leading white space " <a" is needed. But since it's html, it won't display an extra white space in this case.

Reply:
Now updated to open links in the same window.
And that is correct about the whitespace - now removed!